### PR TITLE
Update mmcv version dependence

### PR DESCRIPTION
**Pull request by ly015**

### CODE REVIEW

**Comment by innerlee** [comment]
CI fails

**Review by innerlee — APPROVED** [review]

**Comment by innerlee** [comment]
Change torch 1.5.1 to 1.5.0

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -83,23 +84,17 @@ jobs:
...
-          - torch: 1.7.0+cu101
-            torchvision: 0.8.1+cu101
-            mmcv: "cu101/torch1.7.0"
+          - torch: 1.5.1
+            torchvision: 0.6.1
+          - torch: 1.6.0
+            torchvision: 0.7.0
+          - torch: 1.7.0
+            torchvision: 0.8.1
+          - torch: 1.7.0
```
Comment (innerlee) [inline .github/workflows/build.yml:93]: 1.7.0 is repeated

File: mmpose/__init__.py [hunk 2]
```diff
@@ -3,8 +3,8 @@
 
 from .version import __version__, short_version
 
-mmcv_minimum_version = '1.1.3'
-mmcv_maximum_version = '1.3'
+mmcv_minimum_version = '1.2.6'
```
Comment (innerlee) [inline mmpose/__init__.py:6]: Why is the minimum version changed

Comment (ly015) [reply mmpose/__init__.py:6]: I followed https://github.com/open-mmlab/mmaction2/pull/769/files

Comment (ly015) [reply mmpose/__init__.py:6]: So this should not be changed?

Comment (innerlee) [reply mmpose/__init__.py]: Yeah keep the original one until someone reports incompaitibility

File: .github/workflows/build.yml (hunk 1 above)
Comment (innerlee) [reply .github/workflows/build.yml:93]: okay python version is different

**Pull request merged**